script/tasks/diamondUpdatePeriphery.sh

```
@@ -149,11 +149,11 @@ function diamondUpdatePeriphery() {
 }
 
 register() {
-  local NETWORK=$(tr '[:lower:]-' '[:upper:]_' <<<$1)
+  local NETWORK="$1"
   local DIAMOND=$2
   local CONTRACT_NAME=$3
   local ADDR=$4
-  local RPC="ETH_NODE_URI_$NETWORK"
+  local RPC_URL=$(getRPCUrl $NETWORK)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)